src/main/java/org/acra/ACRAConfiguration.java

```diff
@@ -320,6 +321,12 @@ public ACRAConfiguration setResDialogNegativeButtonText(int resId) {
         return this;
     }
 
+    @SuppressWarnings( "unused" )
+    public ACRAConfiguration setReportDialogClass(Class<? extends CrashReportDialog> reportDialogClass) {

```

Comment:
Well, this was my concern with the whole idea of introducing a CustomDialog originally. 

How do you ensure that the CustomDialog does the appropriate things wrt capturing and sending the report once the user input has been gleaned?